src/views/ObservationDetail.vue

```diff
@@ -0,0 +1,159 @@
+              <td>{{ observation.state }}</td>
+              <td>{{ observation.priority }}</td>
+              <td>{{ observation.modified | formatDate }}</td>
+              <td>{{ observation.created | formatDate }}</td>
+            </tr>
+          </tbody>
+        </table>
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th colspan="4" style="border-right: 2px solid black">Config Status</th>
+              <th colspan="6">Summary</th>
+            </tr>
+            <tr>
+              <th>Id</th>
+              <th>Instrument</th>
+              <th>Guider</th>
+              <th style="border-right: 2px solid black">State</th>
+              <th>State</th>
+              <th>Start</th>
+              <th>End</th>
+              <th>Reason</th>
+              <th>Time</th>
+              <th>Events</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="c_status in observation.configuration_statuses" :key="c_status.id">
+              <td>{{ c_status.id }}</td>
+              <td>{{ c_status.instrument_name }}</td>
+              <td>{{ c_status.guide_camera_name }}</td>
+              <td style="border-right: 2px solid black">{{ c_status.state }}</td>
+              <template v-if="c_status.summary">
+                <td>{{ c_status.summary.state }}</td>
+                <td>{{ c_status.summary.start | formatDate }}</td>
+                <td>{{ c_status.summary.end | formatDate }}</td>
+                <td>{{ c_status.summary.reason }}</td>
+                <td>{{ c_status.summary.time_completed }}</td>
+                <td><a href="#" @click="openEvents(c_status.summary.events)">View</a></td>
+              </template>
+              <template v-else>
+                <td colspan="6">No summary</td>
+              </template>
+            </tr>
+          </tbody>
+        </table>
+      </template>
+    </div>
+  </div>
+</template>
+<script>
+import $ from 'jquery';
+
+import NotFound from '@/components/NotFound.vue';
+import { formatDate } from '@/utils.js';
+
+export default {
+  name: 'ObservationDetail',
+  components: {
+    NotFound
+  },
+  filters: {
+    formatDate(value) {
+      return formatDate(value);
+    }
+  },
+  data: function() {
+    return {
+      id: this.$route.params.id,
+      observation: {},
+      observationLoadError: false,
+      dataLoaded: false
+    };
+  },
+  computed: {
+    observationPortalApiUrl: function() {
+      return this.$store.state.urls.observationPortalApi;
+    }
+  },
+  created: function() {
+    this.getObservation();
+  },
+  methods: {
+    getObservation: function() {
+      let that = this;
+      $.ajax({
+        url: this.observationPortalApiUrl + '/api/observations/' + this.id
+      })
+        .done(function(response) {
+          that.observation = response;
+        })
+        .fail(function(response) {
+          if (response.status !== 404) {
```

Comment:
Does this just happen if the observation doesn’t exist? If so maybe we should say that in the error message above

Reply:
This is setting `observationLoadError` to true for anything that is not a 404, which results in the message saying there was a problem getting the data. For a 404, the `NotFound` component ends up being displayed.